crates/op-rbuilder/Cargo.toml

```diff
@@ -1,9 +1,12 @@
 [package]
-name = "op-rbuilder2"
+name = "op-rbuilder"
 version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+# workspace
+rbuilder = { path = "../rbuilder", features = ["optimism"] }
```

Comment:
Please do not depend on rbuilder for now on this. It is okay to copy/paste code. I want to limit the number of moving parts right now. If we were to depend on something from rbuilder lets do a shared crate and depend there.

Reply:
I'm thinking of a shared crate but imported due to time constraints 👍 I'll copy paste over the code